README.md

```diff
@@ -1,2 +1,90 @@
-# coverter-go
-Convert AsyncAPI documents from older to newer versions.
+# AsyncAPI Converter
+
+Convert AsyncAPI documents from version 1.x to 2.0.0-rc1.
+
+## Installation
+
+`go get github.com/asyncapi/converter-go`
+
+## Usage
+
+### From CLI
+
+```bash
+Usage:
+asyncapi-converter <path> [--toYAML] [--id=<id>]
+asyncapi-converter -h | --help | --version
+
+Options:
+--toYAML    produces results in yaml format
+--id        allows to specify application id
+```
+
+Minimal example:
+
+```bash
+$ asyncapi-converter https://git.io/fjMPF --toYAML
+asyncapi: 2.0.0-rc1
+channels:
+    /:
+        publish:
+            message:
+                oneOf:
+                  - $ref: '#/components/messages/chatMessage'
+...
+```
+
+Specify the application id:
+
+```bash
+$ asyncapi-converter streetlights.yml --id=urn:com.asynapi.streetlights
+{"asyncapi":"2.0.0-rc1","channels":{"/":{"publish":{"message":{"oneOf":[{"$ref":"#/components/messages/chatMessage"}...
+..."id":"urn:com.asynapi.streetlights","info":{"title...
+```
+
+Save the result in a file:
+```bash
+$ asyncapi-converter streetlights.json --toYAML --id=urn:com.asynapi.streetlights > streetlights2.yml
+```
+
+### As a package
+
```

Comment:
Add an introductory sentence to this section describing this way of using the AsyncAPI Converter, please.